lib/server/idemix/nonce.go

```diff
@@ -49,14 +49,19 @@ type NonceManager interface {
 	CheckNonce(nonce *fp256bn.BIG) error
 	// SweepExpiredNonces removes expired nonces from the database
 	SweepExpiredNonces() error
+	// StartNonceSweeper starts a separate thread that will remove expired
+	// nonces at the interval speciifed by the idemix.noncesweepinterval. This
+	// function should be called while initializing the server.
+	StartNonceSweeper()
```

Comment:
Since this is optional (since we conditionally check for nil in the main) it's not a required part of the contract and should not be here.